src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/Controllers/AccountController.cs

```diff
@@ -204,7 +212,9 @@ public ActionResult GoogleLogin(string returnUrl)
         /// <returns> The <see cref="Task"/>. </returns>
         [AllowAnonymous]
         [RequireHttps]
+#pragma warning disable S4261 // Methods should be named according to their synchronicities
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not suppress issues. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1309)